src/java.desktop/unix/classes/sun/java2d/opengl/GLXGraphicsConfig.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2003, 2019, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2003, 2021, Oracle and/or its affiliates. All rights reserved.
```

Comment:
A little optimistic about getting a review between Dec 29th and Dcc 31st perhaps ? ;-)